benchmarks/inclusion/doc/inclusion.md

```diff
@@ -8,57 +8,25 @@ aligns to cell boundaries, leading to much larger difficulties in obtaining an
 accurate representation of the pressure. Specifically, the almost
 discontinuous pressure along this interface leads to oscillations in the
 numerical solution. This can be seen in the visualizations shown in
-Fig.&nbsp;[2][]. As before, for details we refer to (Duretz et al. 2011). The
-analytic solution against which we compare is given in (Schmid and
-Podladchikov 2003). An extensive discussion of convergence properties is given
-in (Kronbichler, Heister, and Bangerth 2012).
+{numref}`fig:inclusion1` and {numref}`fig:inclusion2`. As before, for details we refer to {cite}`DMGT11`. The
+analytic solution against which we compare is given in {cite}`SP03`. An extensive discussion of convergence properties is given in {cite}`kronbichler:etal:2012`.
```

Comment:
I would use cite:t here instead of cite (since the text directly refers to the publications. 
```suggestion
{numref}`fig:inclusion1` and {numref}`fig:inclusion2`. As before, for details we refer to {cite:t}`DMGT11`. The
analytic solution against which we compare is given in {cite:t}`SP03`. An extensive discussion of convergence properties is given in {cite:t}`kronbichler:etal:2012`.
```